programmers/20210506/K번째수.js

```diff
@@ -0,0 +1,8 @@
+function solution(array, commands) {
+    let answer = []
+    for (let i = 0; i < commands.length; i++) {
+        let arr = array.slice(commands[i][0] - 1, commands[i][1]).sort()
```

Comment:
배열은 const로 선언해주시는게 좋아요~
테스트 케이스에서 실패하신 이유는 sort() 메소드 때문이랍니다.
sort는 기본적으로 문자열의 유니코드 포인트를 따릅니다.
그렇기 때문에 ``` [1, 21, 30, 4, 10000].sort() ```의 결과는
``` [1, 10000, 21, 30, 4] ``` 이렇게 정렬되게 됩니다. 그렇기 때문에 수의 크기로 정렬을 해줘야합니다!
```suggestion
        const arr = array.slice(commands[i][0] - 1, commands[i][1]).sort((a, b) => a - b);
```